jellyfin_kodi/objects/tvshows.py

```diff
@@ -377,6 +398,12 @@ def episode_add(self, obj):
             return self.episode_add(obj)
 
         self.jellyfin_db.add_reference(*values(obj, QUEM.add_reference_episode_obj))
+
+        parentPathId = self.jellyfin_db.get_episode_kodi_parent_path_id(*values(obj, QUEM.get_episode_kodi_parent_path_id_obj))
+        if obj['PathId'] != parentPathId:
```

Comment:
```suggestion
        if parentPathId and obj['PathId'] != parentPathId:
```

Along with the return change from the queries, ensure the path actually exists before trying to use it.  It _shouldn't_ be an issue currently since the implicit return is `None`, but just an extra layer of security in case of weirdness happening.  And I would say necessary if we `return 0` instead.